src/main/java/srijana/workshop_1/Task_2.java

```diff
@@ -0,0 +1,22 @@
+
+//Write a short method, which takes String as an argument and returns it as int.
+// If String cannot be converted to int, display this information with System.out.println() to user
+
+
+package srijana.workshop_1;
+
+
+public class Task_2 {
+
+    public  static  void main (String[]args){
+
+        System.out.println(ConvertStringToInt("1234"));
+    }
+
+    public static int ConvertStringToInt(String str ){
+
```

Comment:
Main part is OK, but pay attention to the second part of task:
If String cannot be converted to int, display this information with System.out.println() to user

What if System.out.println(ConvertStringToInt("zzzzzzzzz")) will be called?